docs/ops/normalization/MVN_6.md

```diff
@@ -12,11 +12,14 @@
 \f[
 o_{i} = i_{i} - ReduceMean(i_{k}, axes)
 \f]
-If *normalize_variance* is set to `true`, the output blob is divided by variance. In the case of `eps_mode` equal to `inside_sqrt`:
+
+If *normalize_variance* is set to `true`, the output blob is divided by variance.
```

Comment:
When normalizing the value, the number `eps` is added to the variance to avoid division by zero. According to the `eps_mode` flag's value, `eps` is added inside or outside sqrt:
* If `eps_mode` is `inside_sqrt`: